### PR TITLE
Enable Interaction Model test in Cirque

### LINKED ISSUE #4282: Enable integration tests for IM
#### Problem
We have not yet enabled integration tests for IM Command
 #### Proposed Solution
Enable integration test for IM command

### PR DESCRIPTION
**Pull request by yunhanw-google**
Problem
We don't have e2e validation for Interaction Model.

Summary of Changes
-- Add e2e validation test for Interaction Mode in cirque

Fixes #4282

### CODE REVIEW

**Review by saurabhst — APPROVED** [review]

File: src/app/tests/integration/entrypoint.sh [hunk 1]
```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+service dbus start
+sleep 1
```
Comment (andy31415) [inline src/app/tests/integration/entrypoint.sh:4]: could we remove this sleep (ideally there should be no sleeps, but not sure if possible)? unsure if avahi-daemon and otbr depend on dbus.

**Review by andy31415 — APPROVED** [review]

File: src/app/tests/integration/entrypoint.sh (hunk 1 above)
Comment (yunhanw-google) [reply src/app/tests/integration/entrypoint.sh]: just remove that sleep! Done, thanks

File: src/app/tests/integration/entrypoint.sh [hunk 2]
```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
```
Comment (rwalker-apple) [inline src/app/tests/integration/entrypoint.sh:1]: prefer `#!/usr/bin/env bash`, ok?

File: src/test_driver/linux-cirque/InteractionModelTest.sh [hunk 3]
```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env bash
+
+#
+# Copyright (c) 2020 Project CHIP Authors
```
Comment (rwalker-apple) [inline src/test_driver/linux-cirque/InteractionModelTest.sh:4]: might be 2021 by the time this lands :-)

**Review by rwalker-apple — APPROVED** [review]
just nits...

File: src/test_driver/linux-cirque/InteractionModelTest.sh (hunk 3 above)
Comment (yunhanw-google) [reply src/test_driver/linux-cirque/InteractionModelTest.sh]: Done, thanks!

File: src/app/tests/integration/entrypoint.sh (hunk 2 above)
Comment (yunhanw-google) [reply src/app/tests/integration/entrypoint.sh]: Done, thanks!

**Pull request merged**